hircluster.c

```diff
@@ -2892,7 +2892,7 @@ static void *command_post_fragment(redisClusterContext *cc, struct cmd *command,
         key_count = hiarray_n(command->keys);
 
         reply->elements = key_count;
-        reply->element = hi_calloc(key_count, sizeof(*reply));
+        reply->element = hi_calloc(key_count, sizeof(reply));
```

Comment:
If you use an variable in sizeof, it should be the variable you're assigning I think.

```suggestion
        reply->element = hi_calloc(key_count, sizeof(*(reply->element)));
```

Reply:
You're right, that makes more sense. I've updated the commit.
I've grep'ed through the code to check if I should add the additional parentheses, but saw multiple occurrences without them, so I left them out for consistency (`->` has precendence over `*`, so both forms with and without parentheses are equivalent).

Reply:
Good. I wasn't sure about precedence myself, so thanks for looking it up. :)